examples/cro_vars.yaml

```diff
@@ -0,0 +1,21 @@
+---
+
+# clusterresourceoverride-operator role variables
+cro_role_enable: false
+cro_e2e: false
+cro_install_operator: false
+cro_enable_global_secret: false
+ocp_version: "4.14"
```

Comment:
Can we have `cro_` prefix added to the var here?

Reply:
If possible we should read the cluster version and use the `cro_operator_image var` only if provided. No need for another var `ocp_version`.

Reply:
The version var was added for CI job to override the OCP version in the `cro_operator_image`.  Is it fine if we keep this var with the `cro-` prefix ?

Reply:
Added the prefix `cro_` to the OCP version var.